test/big_natural_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Orangeade.Generator.BigNatural.Tests do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+  import Orangeade.Generator.BigNatural, only: [stream: 0]
+
+  test "check 5 first generated elements" do
+    generated =
+      stream()
+      |> Caffeine.Stream.take(5)
```

Comment:
I often say there's a time and place for piping... when it's something like the following:
```Elixir
alpha = 1..100
beta  = Enum.map(alpha, &FizzBuzz.stringify/1)
gamma = Enum.each(beta, &IO.puts/1)
```
Then the pipe is great:
```Elixir
1..100
|> Enum.map(&FizzBuzz.stringify/1)
|> Enum.each(&IO.puts/1)
```
On the other hand a lot of Elixir code I see pipes just once. The formatter doesn't seem to complain either way. You decide what you think is better style. I guess a fair comparison on style would be with method chaining in OO languages.

Reply:
Thanks for the explanation, that makes sens :)